Makefile

```
@@ -35,3 +35,8 @@ python-quality-test:
 
 javascript-quality-test:
 	./node_modules/.bin/eslint ./eox_core/**/*.js*
+
```

```suggestion

upgrade: export CUSTOM_COMPILE_COMMAND=make upgrade
```
In order to get the generated .txt files have this instead of the `pip-compile` complete command.

---

Updated to the last version on the cookiecutter